### ISSUE TITLE
React development assets

### ISSUE THREAD

**Issue by tlgimenes** [pikapkg/cdn#10]
Hi Pika People.

First of all, congratulations, you are doing a great work.

I'm not sure if I should post it in here, but, I'm trying to use Pika's CDN for react development with import maps

Since I'm developing with react, it would be cool to download the development assets of react so the React's dev tools work properly. 

I'm forcing the following import map
```
imports: {
   "react": "https://cdn.pika.dev/react/react.development.js",
   "react-dom": "https://cdn.pika.dev/react-dom/react-dom.development.js"
}
```

This works great ! However, when I try to add `react-router-dom`, a huge mess occurs.
By adding the following resolution:
```
imports: {
   "react": "https://cdn.pika.dev/react/react.development.js",
   "react-dom": "https://cdn.pika.dev/react-dom/react-dom.development.js"
   "react-router-dom":  "https://cdn.pika.dev/react-router-dom/react-router-dom.development.js"
}
```

the asset `https://cdn.pika.dev/react-router-dom/react-router-dom.development.js` imports the PRODUCTION version of react, loading react twice and breaking react hooks. 

I can think of two solutions:

1. My lack of knowledge of Pika CDN API. If there is a hidden queyrstring in Pika CDN API that correctly sets the NODE_ENV so I could import the assets with `"https://cdn.pika.dev/react?env=development",` and it should work somehow
2. Using the "scopes" of import maps to directly resolve to the development version of react

Have anyone faced a similar problem before ?

Thanks !

**Comment by tlgimenes** [comment]
Just to complement my issue. I've found [this PR](https://github.com/pikapkg/snowpack/pull/228) in `snowpack` that aims to do the same thing as me. However, I couldn't find how they solve this problem

**Comment by FredKSchott** [comment]
Yup, there's no way to do this today on the CDN, but it's a feature we'll be prioritizing asap.

**Comment by joelmoss** [comment]
Seems this has maybe not been prioritised? Is there a workaround in the mean time?

**Comment by LukeChannings** [comment]
@joelmoss 

JSPM's CDN (which has a nice import map generator [here](https://generator.jspm.io/#Y2VgYGBiDs0rySzJSU1hKEpNTC7RTcnPdTA01zPQM4QIQDkAa4g+miwA)) has ESM development bundles, which I've been using in local development to get fast refresh to work. I swap out the URLs in production.
